alf/bin/train_play_test.py

```diff
@@ -224,7 +227,7 @@ def _test(self,
               extra_train_params=None,
               test_play=True,
               extra_play_params=None,
-              test_perf=False,
+              test_perf=True,
```

Comment:
Is the argument necessary? Seems having `test_perf_func` is enough

Reply:
I think we can just leave it there in case that we want to turn off performance checking to save some CI time.